doc/reference/devices_nic.md

```diff
@@ -250,17 +250,18 @@ The targeted device will vanish from the host and appear in the instance (which
 
```

Comment:
Sorry, a bit late with my review - but there's two other small updates needed in this file.

- The note for `nictype`: `physical` (line 240) still states that you can only use `nictype`, not `network`.
- In the "Available NIC types" section, `physical` must now be moved to the section that supports `nictype` and `network`.

Let me know if you'd prefer me to do the changes, but maybe it's a good exercise. :)

Reply:
I fixing that ASAP